app/templating/metadata_context.py

```diff
@@ -41,3 +40,17 @@ def build_metadata_context_for_survey_completed(session_data):
         metadata_context['ru_name'] = session_data.ru_name
 
     return metadata_context
+
+
+def build_schema_metadata(metadata, schema):
+    schema_metadata = schema.json['metadata']
+    parsed = {metadata_field['name']: json_and_html_safe(metadata[metadata_field['name']])
+              for metadata_field in schema_metadata if metadata_field['name'] in metadata}
+
+    return parsed
```

Comment:
We may not need to do this now as the survey specific metadata was probably only used for piping.